### ISSUE TITLE
How to install Hackerspace.sg Docuprint 3055 printer on Windows 10

### ISSUE THREAD

**Issue by adrianke77** [hackerspacesg/hackerspace.sg#128]
Brief notes as follows on installing Docuprint 3055 printer in Hackerspace on Windows 10:

1) Download driver from

http://onlinesupport.fujixerox.com/processDriverForm.do?ctry_code=SG&lang_code=en&d_lang=en&corp_pid=DP3055&rts=null&model=DocuPrint+3055&type_id=2&oslist=Windows+8.1+64+bit&lang_list=en

2) Run the executable to unzip the driver files. Should be unzipped to a folder in \Downloads\

3) Go to the folder with unzipped files, right click on the .inf file and choose Install

4) Go to Control Panel, Devices and Printers, Add Printer, The printer I want isn't listed, Add a printer using TCP/IP address or hostname, leave it on Autodetect, IP address 192.168.6.2, click Next

5) Choose printer from list. Manufacturer Fuji Xerox, Printer is FX DocuPrint 3055 PCL 6

6) It should say the driver is installed. Restart the system, should be printing fine now.

**Comment by notthetup** [comment]
Does this need to go into the website?

**Comment by gibtang** [comment]
Yup, there were times where people had issue looking for the drivers. Maybe
upload the drivers to our server as the drivers could be removed from Fuji
Xerox website any time

On Mon, Sep 28, 2015 at 1:23 PM, Chinmay Pendharkar <
notifications@github.com> wrote:

> Does this need to go into the website?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hackerspacesg/hackerspace.sg/issues/128#issuecomment-143643646>
> .
>



-- 
Regards
Gibson Tang. Managing Director
Author of Objective-C Memory Management Essentials available at Amazon.com
<http://www.amazon.com/Objective-C-Memory-Management-Essentials/dp/1849697124/ref=sr_1_1?s=books&ie=UTF8&qid=1430727396&sr=1-1&keywords=objective-c+memory+management+essentials>
www.azukisoft.com
+65 96565537
www.gibsontang.com

**Comment by robertsky** [comment]
Just commit the file and push it to the master branch?

**Comment by kaihendry** [comment]
Be good if we just had a postscript printer so we didn't have to go through this crazy closed source hoop jumping.

**Comment by notthetup** [comment]
File? What file?

**Comment by hyperair** [comment]
On Sun, Sep 27, 2015 at 11:24:40PM -0700, Kai Hendry wrote:
> Be good if we just had a postscript printer so we didn't have to go through
> this crazy closed source hoop jumping.

The stock (in Ubuntu) foomatic driver for Docuprint P205 works for this printer
over CUPS though.

-- 
Kind regards,
Loong Jin

**Comment by rolandturner** [comment]
On 09/28/2015 02:55 PM, Chow Loong Jin wrote:

> On Sun, Sep 27, 2015 at 11:24:40PM -0700, Kai Hendry wrote:
> > Be good if we just had a postscript printer so we didn't have to go 
> through
> > this crazy closed source hoop jumping.
>
> The stock (in Ubuntu) foomatic driver for Docuprint P205 works for 
> this printer
> over CUPS though.

That would be useful information for the website too.

- Roland

**Comment by chernjie** [comment]
@adrianke77 @elfgoh can you guys coordinate and merge them into either a page or into this repo
https://github.com/hackerspacesg/printer/tree/master/fujiXerox
